### ISSUE TITLE
Use System R

### ISSUE THREAD

**Issue by ltratt** [softdevteam/warmup_stats#68]
Can we use the system R rather than building our own?

**Comment by snim2** [comment]
Working travis config from @smarr 

```yml
language: python

matrix:
  include:
    # - python: "2.7"
    #   os: linux
    #   sudo: false
    #   addons:
    #     apt:
    #       packages:
    #         - build-essential
    #         - r-base
    #         - gfortran
    - language: generic
      os: osx
      before_install: |
        brew cask uninstall oclint  # a travis issue
        brew update > /dev/null
        brew install gcc zlib
        export CFLAGS="-I$(xcrun --show-sdk-path)/usr/include"
script:
 - ./build.sh
```

**Comment by vext01** [comment]
Well I never. I didn't know Travis did OSX.

**Comment by ltratt** [comment]
Nor did I. We can definitely add that in.

But... should we use the system R? We've discussed this before and not come to a conclusion. Now would be a good time to decide.

**Comment by vext01** [comment]
Like I said in MM last week, I think we should use the system R by default, but ensure that the user can put their own R interpreter in `$PATH` in case of another broken distribution package.

Shouldn't be too difficult.

**Comment by ltratt** [comment]
I think the "let the user choose their own" will just happen by default?

**Comment by vext01** [comment]
In theory.

**Comment by ltratt** [comment]
Unless we hard code in search paths, I don't see how it could not occur.

**Comment by vext01** [comment]
I'm hesitant to say it will work for certain as the script invokes the R package manager, which has given us grief in the past.

**Comment by ltratt** [comment]
@snim2 Can you have a look into this and see if it works and whether anything breaks? Thanks!

**Comment by snim2** [comment]
On it...

**Comment by snim2** [comment]
This was fixed in #70
